### PR TITLE
frontend bug fixes

### PR DESCRIPTION
**Pull request by VitaliiShpital**
What: 

Why:

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: web/satellite/src/components/project/UsageReport.vue [hunk 1]
```diff
@@ -41,9 +41,9 @@
 				<div class="usage-report-container__main-area__footer__report-area">
 					<p>Download Advanced Report</p>
 					<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" @click.prevent="onReportClick">
-						<rect width="40" height="40" rx="4" fill="#E2ECF7"/>
-						<path d="M25.6491 19.2809L21.2192 23.5281C20.891 23.8427 20.3988 24 20.0707 24C19.7425 24 19.2503 23.8427 19.0862 23.5281L14.4922 19.2809C13.8359 18.6517 13.8359 17.8652 14.4922 17.236C14.8204 16.9213 15.1485 16.9213 15.6407 16.9213C15.9689 16.9213 16.4611 17.0787 16.6252 17.3933L18.594 19.1236L18.594 11.4157C18.594 10.6292 19.2503 10 20.0707 10C20.891 10 21.5473 10.6292 21.5473 11.4157L21.5473 19.1236L23.5162 17.236C23.6803 16.9213 24.1725 16.9213 24.5006 16.9213C24.8288 16.9213 25.321 17.0787 25.4851 17.3933C26.1414 17.8652 26.1414 18.809 25.6491 19.2809Z" fill="#2683FF"/>
-						<rect x="11" y="28" width="18" height="2" rx="1" fill="#2683FF"/>
+						<rect class="background" width="40" height="40" rx="4" fill="#E2ECF7"/>
```
Comment (VinozzZ) [inline web/satellite/src/components/project/UsageReport.vue]: Should we remove the inline styling rule `fill="#E2ECF7"` if we are using classes, that way we don't need to use `!important`?

File: web/satellite/src/components/project/UsageReport.vue [hunk 2]
```diff
@@ -41,9 +41,9 @@
...
+						<path class="blue" d="M25.6491 19.2809L21.2192 23.5281C20.891 23.8427 20.3988 24 20.0707 24C19.7425 24 19.2503 23.8427 19.0862 23.5281L14.4922 19.2809C13.8359 18.6517 13.8359 17.8652 14.4922 17.236C14.8204 16.9213 15.1485 16.9213 15.6407 16.9213C15.9689 16.9213 16.4611 17.0787 16.6252 17.3933L18.594 19.1236L18.594 11.4157C18.594 10.6292 19.2503 10 20.0707 10C20.891 10 21.5473 10.6292 21.5473 11.4157L21.5473 19.1236L23.5162 17.236C23.6803 16.9213 24.1725 16.9213 24.5006 16.9213C24.8288 16.9213 25.321 17.0787 25.4851 17.3933C26.1414 17.8652 26.1414 18.809 25.6491 19.2809Z" fill="#2683FF"/>
+						<rect class="blue" x="11" y="28" width="18" height="2" rx="1" fill="#2683FF"/>
```
Comment (VinozzZ) [inline web/satellite/src/components/project/UsageReport.vue]: same as above

File: web/satellite/src/components/project/billing/BillingArea.vue [hunk 3]
```diff
@@ -76,16 +82,49 @@
...
-	@media screen and (max-height: 880px) {
-		.billing-overflow {
+	@media screen and (max-height: 900px) {
+		.billing-container__table-holder {
 			overflow-y: scroll;
-			height: 600px;
+			height: 555px;
+
+			&::-webkit-scrollbar, &::-webkit-scrollbar-track, &::-webkit-scrollbar-thumb {
+				visibility: hidden;
```
Comment (VinozzZ) [inline web/satellite/src/components/project/billing/BillingArea.vue]: nit pick: the format on this line is a little off

File: web/satellite/src/components/project/billing/BillingArea.vue [hunk 4]
```diff
@@ -76,16 +82,49 @@
...
+	}
+
+	@media screen and (max-height: 835px) {
+		.billing-container__table-holder {
+			overflow-y: scroll;
+			height: 495px;
+		}
+	}
+
+	@media screen and (max-height: 815px) {
```
Comment (VinozzZ) [inline web/satellite/src/components/project/billing/BillingArea.vue]: The rules in this block is the same as the one above?

File: web/satellite/src/components/project/billing/BillingArea.vue [hunk 5]
```diff
@@ -76,16 +82,49 @@
...
+	}
+
+	@media screen and (max-height: 795px) {
+		.billing-container__table-holder {
+			overflow-y: scroll;
+			height: 475px;
+		}
+	}
+
+	@media screen and (max-height: 780px) {
```
Comment (VinozzZ) [inline web/satellite/src/components/project/billing/BillingArea.vue]: The rules in this block is the same as the one above?

Comment (VitaliiShpital) [reply web/satellite/src/components/project/billing/BillingArea.vue]: Thanks for review! Changes applied

File: web/satellite/src/components/account/billing/BillingArea.vue [hunk 6]
```diff
@@ -59,38 +61,49 @@
...
 	@media screen and (max-height: 795px) {
-		.billing-overflow {
+		.billing-container__table-holder {
 			overflow-y: scroll;
-			height: 640px;
+			height: 475px;
 		}
 	}
 
 	@media screen and (max-height: 780px) {
```
Comment (VinozzZ) [inline web/satellite/src/components/account/billing/BillingArea.vue]: Shouldn't this block inherent the rules from line 96(https://github.com/storj/storj/pull/2127/files#diff-06d1d46adf3531fcd70bc6e4287c9c96R96)?

Comment (VitaliiShpital) [reply web/satellite/src/components/account/billing/BillingArea.vue]: Yes, sure. Done!

**Pull request merged**